experiments/stream/experiment.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from benchpark.error import BenchparkError
+from benchpark.directives import variant
+from benchpark.experiment import Experiment
+from benchpark.expr.builtin.caliper import Caliper
+
+
+class Stream(
+    Experiment,
+    Caliper,
+):
+    variant(
+        "workload",
+        default="stream",
+        description="stream",
+    )
+
+    variant(
+        "version",
+        default="5.10",
+        description="app version",
+    )
+
+    def compute_applications_section(self):
+        # TODO: Replace with conflicts clause
+        scaling_modes = {
+            "single_node": self.spec.satisfies("+single_node"),
+        }
+
+        scaling_mode_enabled = [key for key, value in scaling_modes.items() if value]
+        if len(scaling_mode_enabled) != 1:
+            raise BenchparkError(
+                f"Only one type of scaling per experiment is allowed for application package {self.name}"
+            )
```

Comment:
@august-knox We can remove this. single_node variant is always enabled by default